integrate-kobiton-teamcity.md

```diff
@@ -0,0 +1,98 @@
++ [B. Setup](#b-setup)
+  + [1.Preparing automation test repository](#1-preparing-automation-test-repository)
+  + [2. Getting desired capabilities](#2-getting-desired-capabilities)
+  + [3. Configuring automation test script](#3-configuring-automation-test-script)
+  + [4. Configuring TeamCity project](#4-configuring-teamcity-project)
++ [C. Execution](#c-execution)
+  + [1. Runnig automation test on Kobiton devices](#1-running-automation-test-on-kobiton-devices)
+  + [2. Fetch test session data through Kobiton REST API](#2-fetch-test-session-data-through-kobiton-rest-api)
++ [D. Feedback](#d-feedback)
+## A. PREREQUISTIES 
+- Kobiton account: Go to https://portal.kobiton.com/register to create new account.
+- Kobiton Username and API Key: Folow instruction at `IV. Configure Test Script for Kobiton` section on [our blog article](https://kobiton.com/blog/tutorial/parallel-testing-selenium-webdriver/) to get Username and API Key. 
+## B. SETUP
+### 1. Preparing automation test repository
+To perform automation testing on Kobiton, you need a GitHub repository fully configured with TeamCity. We have provided a sample script for executing automation test with Kobiton.
+
+To use the provided sample, follow these step:
++ Clone [this repository](https://github.com/kobiton/TeamCity-HockeyApp-Appium-ReactNative).
++ Create a new GitHub repository.
++ Copy contents of `sample` folder into your repository and create a project in TeamCity with this repository URL.
+### 2. Getting desired capabilities
+Desired capabilities need to be added to the automation test script to allow the test to be executed on Kobiton device(s).
+  + Click **Devices** in the top navigation bar.
+
+    ![click devices](./assets/devices.png)
+ 
+  + Hover over any device you want to test with and click on Automation settings button (the one with gear symbol).
+ 
+    ![click auto](./assets/click_auto.png)
+ 
+  + On the left-hand side, you can select your preferred language, App Type, etc.
+  In this example, we use **NodeJS** as the default language of the script. Therefore, choose **NodeJS** in `Language` section and **Hybrid/Native from Apps** in `App type` section. On the right-hand side, Kobiton automatically generates your desired capabilities into code based on the options you selected. Copy the code to prepare for the next step.
+
+    ![click auto](./assets/automation.png) 
+### 3. Configuring automation test script
+
+Open automation test script in your repository.
+
+Replace `desiredCaps` in the script with one collected in the previous step.
+
+Example:
+
+```javascript
+const desiredCaps = {
+  sessionName:        'Automation test session',
+  sessionDescription: '', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: true, 
+  app:                '<APP_URL>', 
+  deviceGroup:        'KOBITON', 
+  deviceName:         '<DEVICE_NAME>',
+  platformVersion:    '<DEVICE_VERSION>',
+  platformName:       '<DEVICE_PLATFORM_NAME>' 
+}
+```
+
```

Comment:
We should describe a little bit parameters (which is required or not):
* YOUR_APP_URL: (app repo or external link)
* DEVICE_NAME
* PLATFORM_VERSION
* PLATFORM_NAME

You should provide some more documentation to describe clearer about parameters, like [this](https://github.com/kobiton/CircleCI-HockeyApp-Appium-ReactNative/blob/0406b8d1f1d94dd2e8f9367e22b3ac16a7678c1d/Integrate-Kobiton.md#22-setting-desired-capabilities).